launcher/launcher.go

```diff
@@ -23,12 +24,22 @@ type Launcher interface {
 	SetLogFile(logFile string)
 }
 
+// OutputHandler is invoked with the read end of a pipe to which the Launcher forwards
+// output (stdout or stderr). Each instance of an OutputHandler will be provided its own
+// goroutine, so it may block as necessary.
+type OutputHandler = func(pipe io.ReadCloser)
+
 type Options struct {
 	IsRunningFromBrowser          bool
 	JavaDir                       string
 	ShowConsole                   bool
 	DisableVerification           bool
 	DisableVerificationSameOrigin bool
+
+	// If non-nil, processes output from stdout of the launched process
+	StdoutHandler OutputHandler
```

Comment:
An alternative to this would be to add it to the interface. I chose the options to limit the requirements of the interface, but I'm good with either approach.